easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -83,6 +84,10 @@ def configure_step(self, srcdir=None, builddir=None):
                 srcdir = default_srcdir
 
         options = ['-DCMAKE_INSTALL_PREFIX=%s' % self.installdir]
+
+        if self.cfg.get('releasebuild', True):
+            options.append("-DCMAKE_BUILD_TYPE=Release")
```

Comment:
wouldn't it be nicer to check whether `-DCMAKE_BUILD_TYPE` is already specified via `self.cfg['configopts']`?

```python
if self.cfg.get('releasebuild', True) and '-DDCMAKE_BUILD_TYPE' not in self.cfg['configopts']:
    options.append("-DCMAKE_BUILD_TYPE=Release")
```

in that case, you can even drop the `releasebuild` parameter?

also, it would be nicer to support `build_type = 'Release'` instead anyway?